src/components/Hero/Hero.module.scss

```diff
@@ -13,4 +17,56 @@
   transition: transform ease 1.25s, opacity ease 1.5s;
   opacity: 0;
   transform: translateY(80px);
-}
\ No newline at end of file
+}
+
+.intro {
+  flex-direction: column;
+  height: 100vh;
+  
+  }
+
+.fadeInSubheadline {
+  opacity: 0;
+  transition: opacity 4s;
+  &:global(.show) {
+    opacity: 1;
+  }
+}
+
+.headline {
+  text-transform: uppercase;
+  transition: transform 0.85s ease;
+  
+  &::before {
+    background: #fff;
+    content: "";
+    display: block;
+    height: 2px;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 0%;
+    transform: scale(0);
+      }
+    
+  &::after {
+    background: #fff;
+    bottom: 0;
+    content: "";
+    display: block;
+    height: 2px;
+    position: absolute;
+    right: 0;
+    width: 0%;
+    transform: scale(0);
```

Comment:
Do we need this?

Reply:
Nope the scale is not necessary. I removed it.